src/exoplanet_core/pymc/__init__.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+
+__all__ = ["ops"]
+
+
+def __set_compiler_flags():
+    import pytensor
+
+    def add_flag(current, new):
+        if new in current:
+            return current
+        return f"{current} {new}"
+
+    current = pytensor.config.gcc__cxxflags
+    current = add_flag(current, "-Wno-c++11-narrowing")
+    current = add_flag(current, "-fno-exceptions")
+    current = add_flag(current, "-fno-unwind-tables")
+    current = add_flag(current, "-fno-asynchronous-unwind-tables")
+    pytensor.config.gcc__cxxflags = current
```

Comment:
I'm not sure that we'll need these compiler flags for the current version - these were mostly hacks for theano issues!

Reply:
If the tests still pass without these flags, does it mean they can be removed? Otherwise what would be the best way to check that they are still needed?

Reply:
PyMC have [something similar](https://github.com/pymc-devs/pymc/blob/602234b98d7c2c1e4382a56c16e7f71c1241bc8a/pymc/__init__.py#L28-L47) in their `__init__.py`:

```python
def __set_compiler_flags():
    # Workarounds for PyTensor compiler problems on various platforms
    import pytensor


    current = pytensor.config.gcc__cxxflags
    augmented = f"{current} -Wno-c++11-narrowing"


    # Work around compiler bug in GCC < 8.4 related to structured exception
    # handling registers on Windows.
    # See https://gcc.gnu.org/bugzilla/show_bug.cgi?id=65782 for details.
    # First disable C++ exception handling altogether since it's not needed
    # for the C extensions that we generate.
    augmented = f"{augmented} -fno-exceptions"
    # Now disable the generation of stack unwinding tables.
    augmented = f"{augmented} -fno-unwind-tables -fno-asynchronous-unwind-tables"


    pytensor.config.gcc__cxxflags = augmented




__set_compiler_flags()
```

Reply:
Or would it be better to keep them as long as PyMC does? Given that it's possible to use the PyTensor backend without importing PyMC (technically at least, in practice everyone probably uses it with PyMC)